bower.json

```diff
@@ -22,14 +22,15 @@
   ],
   "dependencies": {
     "angular": "~1.4",
-    "angular-sanitize": "~1.4",
     "angular-animate": "~1.4",
-    "angular-ui-router": "~0.2.10",
     "angular-bootstrap": "1.3",
+    "angular-i18n" : "1.6.4",
```

Comment:
> We'd like to get rid of the `bower` dependency and install everything through `npm`. When you add new frontend dependencies, please try to declare them in `package.json` rather than `bower.json`.
> In this specific case, as an Angular dependency, it might indeed be more pertinent here so that dependency constraints can be solved properly.